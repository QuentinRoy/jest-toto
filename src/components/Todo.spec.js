import React from 'react'
import renderer from 'react-test-renderer'
import Todo from './Todo'

describe('<Todo />', () => {
  it('renders correctly when not completed', () => {
    const tree = renderer.create(
      <Todo
        completed={false}
        text="foo"
        onClick={()=>{}}
      />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('renders correctly when completed', () => {
    const tree = renderer.create(
      <Todo
        completed
        text="bar"
        onClick={()=>{}}
      />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
