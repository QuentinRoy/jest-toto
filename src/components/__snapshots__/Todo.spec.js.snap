exports[`<Todo /> renders correctly when completed 1`] = `
<li
  onClick={[Function]}
  style={
    Object {
      "textDecoration": "line-through",
    }
  }>
  bar
</li>
`;

exports[`<Todo /> renders correctly when not completed 1`] = `
<li
  onClick={[Function]}
  style={
    Object {
      "textDecoration": "none",
    }
  }>
  foo
</li>
`;
